index.rst

```diff
@@ -1,61 +1,30 @@
-..
-  Technote content.
-
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
-
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
-
-  Use the following syntax for sections:
-
-
-
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
-
-  .. figure:: /_static/filename.ext
-     :name: fig-label
-
-     Caption text.
-
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
-
-   Feel free to delete this instructional comment.
 
 :tocdepth: 1
 
 .. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
 .. sectnum::
 
-.. TODO: Delete the note below before merging new content to the master branch.
+.. TODO: EDIT ts_standardscripts and other links to point to new repo locations upon completion of renaming
 
-.. note::
-
-   **This technote is still being drafted and has not yet been accepted as the official workflow.**
-
-.. Do not include the document title (it's automatically added from metadata.yaml).
 
 Introduction
 ===============
-This technote is to detail out the observing jobs (previously known as scripts or SAL scripts)
-and notebook development cycle from a simple test
+This technote is to detail out the observing scripts and notebook development cycle from a simple test
 or idea, developed inside a notebook, to evolving into a method that could be called by other
-notebooks, and/or into a Job to be called via the Queue, then finally to the level of being a sanctioned and
-regularly maintained Job as part of operations.
+notebooks, and/or into a script to be called via the scriptQueue, then finally to the level of being a sanctioned and
+regularly maintained script as part of operations.
 
 Documented below are the details for each step include which repos should be used for the different aspects of
 development and the level of
 testing/review required at each stage. The note is built upon a sample use-case which was encountered early on in the
-commissioning phase of the AuxTel, where a user wants to derive a high-level piece of functionality which requires
-new additions to code at multiple levels to implement as a final production Job. In short, the use-case is:
+commissioning phase of the Auxiliary Telescope, where a user wants to derive a high-level piece of functionality which
```

Comment:
Can we break this phrase up? I think it is too long. 

I would suggest something on these lines:

> The note is built upon a sample use-case which was encountered early on in the commissioning phase of the Auxiliary Telescope ~, where~ **. In this case,** a user wants to derive a high-level piece of functionality ~which~ **that** requires new ~additions to code~ **software** at multiple levels to **allow the** ~implement as~ **implementation of** a final production ~script~ `Script`.

I think we should start calling "Scripts" with capital "S" when we talk about scripts that goes into the `ScriptQueue`? We could also format them as code to suggest an entity, e.g., `Script`